design-system/materials/internals/definition.yaml

```diff
@@ -1030,6 +1052,8 @@ decisionGroupsByTheme:
           choice: font-weight-400
         font-weight-for-table-header:
           choice: font-weight-400
+        font-weight-for-text-when-bold:
+          choice: 'bold'
```

Comment:
Could we maybe use `font-weight-700` token here? (it's equivalent to `bold`: [reference](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#syntax))

Reply:
Updated [5da68eb](https://github.com/commercetools/ui-kit/pull/2452/commits/5da68ebc379dd3668abb4a5470229a9709e945da)